src/components/pages/units-page/edit-unit-dialog.tsx

```diff
@@ -375,6 +352,19 @@ export const EditUnitDialog = (props: EditUnitDialogProps) => {
                 <TableBody>
                   {musterConfiguration.map(muster => (
                     <TableRow key={muster.rowKey}>
+                      <TableCell className={classes.reportCell}>
+                        <Select
```

Comment:
Do we care that the styling of the dropdown doesn't match other `select` controls?